build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml

```diff
@@ -0,0 +1,36 @@
+parameters:
+# Extract the build revision number from Build.BuildNumber. This is needed to pass to build-nupkg
+  subversion: '$env:BUILD_BUILDNUMBER.substring($env:BUILD_BUILDNUMBER.length - 3, 3)'
```

Comment:
This makes there be a strong coupling between this script and the build Pipeline that calls it. If the calling Pipeline does not happen to have the expected build naming scheme, this will blow up.

It might be better to have the subversion passed as an explict parameter. It could default to 0 and things would still work.

Reply:
Can do.